docs/DevGuide/Parallelization.md

```diff
@@ -168,8 +168,9 @@ Each %Parallel Component struct must have the following type aliases:
       a node on a supercomputer. However, for non-SMP builds nodes and cores are
       equivalent. We ensure that all entry method calls done through the
       Algorithm's `simple_action` and `receive_data` functions are
-      threadsafe. User controlled threading is possible by calling the non-entry
-      method member function `threaded_action`.
+      threadsafe. User-controlled threading is possible by calling the entry
+      method member function `threaded_action`, which is like `simple_action`
+      except it passes a node lock to the `Action`'s apply function.
```

Comment:
I think this is the only part of this documentation where threaded actions are mentioned, so maybe it is worth also pointing out that threaded actions are the only case where the runtime system permits multiple actions executing simultaneously on the same component? (that's somewhat hinted at here, but not explicitly stated.)